### PR TITLE
Added code for Input & Output in Go

### PR DESCRIPTION
**Pull request by drafailov**
Added code for Input & Output in Go.

### CODE REVIEW

File: cmd/comparegocopy/main.txt [hunk 1]
```diff
@@ -0,0 +1,55 @@
+	//The copy built-in function copies elements from a source slice into a destination slice
```
Comment (mgenov) [inline cmd/comparegocopy/main.txt]: main.txt doesn't look related to the task.

File: gofile/filetest.go [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+		} else {
+			buf = append(buf, '\n')
+			p = append(p, buf...)
+		}
+	}
+
+	return
+}
+
+// ConsoleSource returns a slice of data bytes from the console
```
Comment (mgenov) [inline gofile/filetest.go]: Comment and name of the function are not same.

File: gofile/filetest.go [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+func OpenFile(path string) (file *os.File, err error) {
+	fmt.Print(path)
+	file, err = os.OpenFile(path, os.O_APPEND|os.O_CREATE|os.O_WRONLY, os.FileMode(0644))
+	if err != nil {
+		return
+	}
+	return
+}
+
+// scaner returns a byte slice containing the bytes from the given reader.
```
Comment (mgenov) [inline gofile/filetest.go]: Scanner reads content from the provided Reader till it reaches the passed breaker.

**Comment by drafailov** [comment]
Renamed func ConsoleScanner to func BreakableConsoleScanner
